volttron/platform/instance_setup.py

```diff
@@ -268,7 +268,10 @@ def func(*args, **kwargs):
             _shutdown_platform()
 
             if identity is not None:
-                os.environ.pop('AGENT_VIP_IDENTITY')
```

Comment:
This will be in develop shortly after the 8.1.1 release this week.

Reply:
Thanks for that.